pages/installation/virtual_machine_appliances.rst

```diff
@@ -55,7 +56,7 @@ Logging in
 
 You can log into the shell of the operating system of the appliance with the
 user *ubuntu* and the password *ubuntu*. You should of course change those
-credentials if you plan to go into production with the appliance.
+credentials if you plan to go into some kind of production with the appliance.
```

Comment:
How about removing the last part and just leave the sentence at

> You should change those credentials!

The "some kind of production" doesn't make much sense to me.